templates/environment/versions/cf-v1.4.0.yml

```diff
@@ -0,0 +1,376 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+Description: CloudFormation environment template for infrastructure shared among Copilot workloads.
+Metadata:
+  Version: 'v1.4.0'
+Parameters:
+  AppName:
+    Type: String
+  EnvironmentName:
+    Type: String
+  ALBWorkloads:
+    Type: String
+    Default: ""
+  EFSWorkloads:
+    Type: String
+    Default: ""
+  NATWorkloads:
+    Type: String
+    Default: ""
+  ToolsAccountPrincipalARN:
+    Type: String
+  AppDNSName:
+    Type: String
+    Default: ""
+  AppDNSDelegationRole:
+    Type: String
+    Default: ""
+  Aliases:
+    Type: String
+    Default: ""
+Conditions:
+  CreateALB:
+    !Not [!Equals [ !Ref ALBWorkloads, "" ]]
+  DelegateDNS:
+    !Not [!Equals [ !Ref AppDNSName, "" ]]
+  ExportHTTPSListener: !And
+    - !Condition DelegateDNS
+    - !Condition CreateALB
+  CreateEFS:
+    !Not [!Equals [ !Ref EFSWorkloads, ""]]
+  CreateNATGateways:
+    !Not [!Equals [ !Ref NATWorkloads, ""]]
+  HasAliases:
+    !Not [!Equals [ !Ref Aliases, "" ]]
```

Comment:
is the new parameter and the condition the only diff?

Reply:
yeah